### DISCUSSION TITLE
Is there a alt-token limit?

### DISCUSSION THREAD

**Discussion by Asvarius** [Vedza/NitroSniperGo#267] [Q&A]
So, I was creating a bunch of accounts, now I got like 20, all of them with 40+ servers. when I restarted in the log said I had only 138 servers. Is this just printing the wrong number? how can I solve?
obs: I verified if all tokens were written corretly

**Comment by Hass1000** [comment]
yeah its a bug, for some reason is prints the wrong amount, just it ignore it.
